src/components/v5/common/TableWithBurgerMenu/TableWithBurgerMenu.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { createColumnHelper } from '@tanstack/react-table';
+import BurgerMenu from '~v5/common/Table/partials/BurgerMenu';
+import Table from '~v5/common/Table';
+import useToggle from '~hooks/useToggle';
+import { TableWithBurgerMenuProps } from '../Table/types';
+
+const displayName = 'v5.common.TableWithBurgerMenu';
+
+const TableWithBurgerMenu = <T,>({
+  className,
+  tableTitle,
+  columns,
+  actions,
+}: TableWithBurgerMenuProps<T>) => {
+  const columnHelper = createColumnHelper();
+  const [selectedRowId, setSelectedRowId] = useState<string>();
+  const [
+    isMenuVisible,
+    { toggle: onToogle, toggleOff: onToogleOff, registerContainerRef },
+  ] = useToggle();
+  const { fields, append, remove, getValues } = actions;
+
+  const burgerColumn = [
+    columnHelper.accessor('menu', {
+      header: () => '',
+      // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
I think that you shouldn't leave it like this

Reply:
I haven't found any other way then that.

Reply:
Maybe it's used in a wrong way